README.md

```diff
@@ -555,7 +554,7 @@ const okta = require('@okta/okta-sdk-nodejs');
 const MemoryStore = require('@okta/okta-sdk-nodejs/src/memory-store');
 
 const client = new okta.Client({
-  orgUrl: 'https://dev-1234.oktapreview.com/'
+  orgUrl: 'https://dev-1234.oktapreview.com/',
```

Comment:
This looks like it too should be `{yourOktaDomain}`

Reply:
Unfortunately the README has references to both `{yourOktaDomain}` and `https://dev-1234.oktapreview.com/` 
I think it's better we start removing all references to `{yourOktaDomain}` as a dummy org name is clearer.